### PR TITLE
HOUSNAV-30: landing page UI

### PR DESCRIPTION
**Pull request by nicholaslyon-dt**
[HOUSNAV-30](https://hous-bssb.atlassian.net/browse/HOUSNAV-30)

## Overview & Purpose
Update the landing page to look like the design plus some tweaks - minus the footer. That is a separate ticket.

## Summary of Changes
- Created a page stylesheet for the landing page
- Updated the header image to match the bc.gov site
- Added a placeholder footer

## Testing
Check out the landing page at all screen sizes

## Checklist
- [x] I have written or updated vitests for this work
- [x] I have reviewed the [accessibility guidelines](https://digital.gov.bc.ca/wcag/home/) relevant to this work
- [x] I have reviewed this work with at least one screen reader (when applicable)
- [x] I have added/updated any related documentation

### CODE REVIEW

File: apps/web/app/page.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+.page-Root--WalkthroughCards li {
+  display: flex;
+}
+
+.page-Root--PreFooter {
+  background-color: var(--theme-gray-110);
+  color: var(--theme-gray-10);
+  font: var(--typography-regular-small-body);
+  border-top: var(--layout-border-width-large) solid var(--theme-primary-gold);
+  border-bottom: var(--layout-border-width-large) solid
```
Comment (wardm89) [inline apps/web/app/page.css:42]: ` --layout-border-width-medium` looks better to me. The 4px yellow I was kinda straining my eyes.

**Review by wardm89 — APPROVED** [review]
LGTM but would consider reducing that border width.

File: apps/web/app/page.css (hunk 1 above)
Comment (nicholaslyon-dt) [reply apps/web/app/page.css:42]: It's 4 to match what the [BC Gov Homepage](https://www2.gov.bc.ca/gov/content/home) has it set as. 🤷🏻

Comment (wardm89) [reply apps/web/app/page.css:42]: I see but yeah it like is "blinding" to me to have the yellow so bold next to the white.

**Pull request merged**